projects/netgrif-components/src/lib/admin/user-invite/user-invite.component.scss

```diff
@@ -17,6 +17,7 @@
     width: 100%;
     white-space: nowrap;
     border: 1px solid #eeeeee;
+    background: white;
```

Comment:
yes, but the current color was transparent, so when you put there some another color as white ( maybe grey) , it was hardly to read the text , this would fix only color of cards in this components